app/code/Magento/Indexer/Test/Mftf/ActionGroup/IndexerActionGroup/UpdateIndexerByScheduleActionGroup.xml

```diff
@@ -7,7 +7,7 @@
 -->
 <actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
-    <actionGroup name="updateIndexerBySchedule">
+    <actionGroup name="UpdateIndexerBySchedule">
```

Comment:
Replace that to `AdminIndexerSetUpdateOnScheduleActionGroup`, also create `ActionGroup/_Deprecated_/updateIndexerOnScheduleActionGroup.xml` with contents:
```xml
<actionGroup name="updateIndexerOnSchedule" extends="AdminIndexerSetUpdateOnScheduleActionGroup" deprecated="Use AdminIndexerSetUpdateOnScheduleActionGroup"/>
```
Please also keep in mind that filename has to be the same name that Action Group's name inside.